content/documentation/pages/6-recipes/6-multi-platform-deployment/1-multiple-platform-accounts.md

```diff
@@ -0,0 +1,531 @@
+metadata:
+  name: skipper
+  labels:
+    app: skipper
+data:
+  application.yaml: |-
+    spring:
+      cloud:
+        skipper:
+          server:
+            platform:
+              kubernetes:
+                accounts:
+                  default:
+                    namespace: default
+                    environmentVariables: 'SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT},SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}'
+                    limits:
+                      memory: 1024Mi
+                      cpu: 500m
+                    readinessProbeDelay: 120
+                    livenessProbeDelay: 90
+                  kafkazone:
+                    namespace: kafkazone
+                    environmentVariables: 'SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS=${KAFKA_SERVICE_HOST}:${KAFKA_SERVICE_PORT},SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES=${KAFKA_ZK_SERVICE_HOST}:${KAFKA_ZK_SERVICE_PORT}'
+                    limits:
+                      memory: 2048Mi
+                      cpu: 500m
+                    readinessProbeDelay: 180
+                    livenessProbeDelay: 120
+      datasource:
+        url: jdbc:mysql://${MYSQL_SERVICE_HOST}:${MYSQL_SERVICE_PORT}/skipper
+        username: root
+        password: ${mysql-root-password}
+        driverClassName: org.mariadb.jdbc.Driver
+        testOnBorrow: true
+        validationQuery: "SELECT 1"
+```
+
+[[note]]
+| Notice that the inclusion of a platform account with the name `kafkazone`. Also, the default memory for the deployed
+| pod is set to 2GB along with readiness and liveness probe customizations.
+
+For batch data pipelines, however, you'd have to change the configurations in `server-config.yaml` as follows.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scdf-server
+  labels:
+    app: scdf-server
+data:
+  application.yaml: |-
+    spring:
+      cloud:
+        dataflow:
+          applicationProperties:
+            stream:
+              management:
+                metrics:
+                  export:
+                    prometheus:
+                      enabled: true
+                      rsocket:
+                        enabled: true
+                        host: prometheus-proxy
+                        port: 7001
+            task:
+              management:
+                metrics:
+                  export:
+                    prometheus:
+                      enabled: true
+                      rsocket:
+                        enabled: true
+                        host: prometheus-proxy
+                        port: 7001
+          grafana-info:
+            url: 'https://grafana:3000'
+          task:
+            platform:
+              kubernetes:
+                accounts:
+                  default:
+                    namespace: default
+                    limits:
+                      memory: 1024Mi
+                  highmemory:
```

Comment:
It would be nice if we change the name of the `account` as it may confuse the users if the name of the `namespace` and the name of the `account` have to be same.

Reply:
Same comment as above. Changing namespace to be more descriptive.